core/src/org/javarosa/core/model/FormDef.java

```diff
@@ -109,10 +109,19 @@
     // arguments to captions
     private Vector outputFragments;
 
-    public Hashtable<TreeReference, Vector<Triggerable>> triggerIndex;
+    /**
+     * Map references to the calculate/relevancy conditions that depend on that
+     * reference's value. Used to trigger re-evaluation of those conditionals
+     * when the reference is updated.
+     */
+    private Hashtable<TreeReference, Vector<Triggerable>> triggerIndex;
```

Comment:
Made this private and exposed methods to access its data in an immutable manner.